useful-scripts/misc-test-utils/src/services/publicationService.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-console */
+import axios from 'axios';
+import chalk from 'chalk';
+import { v4 } from 'uuid';
+import errorHandler from '../utils/errorHandler';
+import { ReleaseData } from '../types/ReleaseData';
+import sleep from '../utils/sleep';
+
+const { ADMIN_URL, TOPIC_ID, JWT_TOKEN } = process.env;
+
+const publicationService = {
+  // eslint-disable-next-line consistent-return
+  createPublication: async () => {
+    console.time('createPublication');
+
+    try {
+      const res = await axios({
+        method: 'POST',
+        url: `${ADMIN_URL}/api/publications`,
+        data: {
+          title: `importer-testing-${v4()}`,
+          topicId: TOPIC_ID,
+          contact: {
+            teamName: 'testing',
+            teamEmail: 'johndoe@gmail.com',
+            contactName: 'John Doe',
+            contactTelNo: '123456789',
+          },
+        },
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${JWT_TOKEN}`,
+        },
+      });
+
+      console.timeEnd('createPublication');
+      console.log(
+        chalk.green(`Created publication. Status code ${res.status}`),
+      );
+      return res.data.id;
+    } catch (e) {
+      errorHandler(e);
+    }
+    return null;
+  },
+  publishRelease: async (obj: ReleaseData, releaseId: string) => {
+    try {
+      await axios({
+        method: 'POST',
+        url: `${ADMIN_URL}/api/releases/${releaseId}/status`,
+        headers: {
+          Authorization: `Bearer ${JWT_TOKEN}`,
+          'Content-Type': 'application/json',
+        },
+        data: obj,
+      });
+    } catch (e) {
+      errorHandler(e);
+    }
+  },
+  getPublicationProgress: async (releaseId: string) => {
```

Comment:
This seems like the wrong name - `getReleaseProgress` might be better?

Also - shouldn't this be in `releaseService`?